### ISSUE TITLE
PNGs corrupted by ascii conversion

### ISSUE THREAD

**Issue by babazaroni** [rlabbe/Kalman-and-Bayesian-Filters-in-Python#253]
Can only view the pngs if they are already rendered in a notebook.  For instance, residual_chart.png in chapter 4 is only viewable on github.  The png's in the fig folder are corrupted and cannot be viewed by standard image viewers, on Mac or Ubuntu.  I suspect they may have been corrupted on upload to github.

**Comment by x1angli** [comment]
I have the same problem here. 

for example, if we go to 
https://github.com/rlabbe/Kalman-and-Bayesian-Filters-in-Python/blob/master/06-Multivariate-Kalman-Filters.ipynb

>  a measurement and a prediction, as in this chart:

then the image immediately following it (which is "./figs/residual_chart.png") is corrupted. 

I checked the .png file, it's corrupted. I can't view it on github, on nbviewer, or on my local machine (I did a git-clone).
